app/react-components/Project.js

```diff
@@ -395,7 +396,33 @@ const Project = ({
                 { key: 'num_mutations', title: '# Mutations'},
                 {
                   title: <i className="fa fa-bar-chart-o"><div style={styles.hidden}>add to survival plot</div></i>,
-                  onClick: (d) => setSurvivalGene(d === survivalGene ? null : d),
+                  onClick: (d) => {
+                    if (d.survivalId !== geneData.id) {
+                      const filters = [
+                        {"op": "=","content": {"field": "genes.symbol","value": d.survivalId}},
+                        {"op": "not","content": {"op": "=","content": {"field": "genes.symbol","value": d.survivalId}}}
+                      ];
+
+                      const tempFilters = [
+                        {"op":"=","content":{"field":"cases.project.project_id","value":"TCGA-KIRC"}},
+                        {"op":"=","content":{"field":"cases.project.project_id","value":"TCGA-KIRP"}}
+                      ];
```

Comment:
Yep, temp filters is used in place of the ICGC endpoint until we get the data in the api. I will add a comment to make it a little clearer why it's there though.